gui/src/containers/Bridges/Edit.js

```diff
@@ -39,53 +39,52 @@ export const Edit = useHooks(props => {
   return (
     <Content>
       <Grid container>
-        <Grid item xs={12}>
-          <Breadcrumb>
-            <BreadcrumbItem href='/'>Dashboard</BreadcrumbItem>
-            <BreadcrumbItem>></BreadcrumbItem>
-            <BreadcrumbItem href='/bridges'>Bridges</BreadcrumbItem>
-            <BreadcrumbItem>></BreadcrumbItem>
-            <BreadcrumbItem>{bridge && bridge.id}</BreadcrumbItem>
-          </Breadcrumb>
-        </Grid>
-        <Grid item xs={12} md={12} xl={6}>
-          <Grid container>
-            <Grid item xs={9}>
-              <Title>Edit Bridge</Title>
-            </Grid>
-            <Grid item xs={3}>
-              <Grid container justify='flex-end'>
-                <Grid item>
-                  {bridge &&
-                    <Button
-                      variant='outlined'
-                      color='primary'
-                      component={ReactStaticLinkComponent}
-                      to={`/bridges/${bridge.id}`}
-                    >
-                      Cancel
-                    </Button>
-                  }
+        <Grid item xs={12} md={11} lg={9}>
+          <Card>
+            <CardContent>
+              <Grid container>
+                <Grid item xs={9}>
+                  <Typography variant='h5' color='secondary'>
+                    Edit Bridge
+                  </Typography>
+                </Grid>
+                <Grid item xs={3}>
+                  <Grid container justify='flex-end'>
+                    <Grid item>
+                      {bridge &&
+                        <Button
+                          variant='outlined'
+                          color='primary'
+                          component={ReactStaticLinkComponent}
+                          to={`/bridges/${bridge.id}`}
+                        >
+                          Cancel
+                        </Button>
+                      }
+                    </Grid>
+                  </Grid>
                 </Grid>
               </Grid>
-            </Grid>
-          </Grid>
+            </CardContent>
+
+            <Divider />
 
-          {onLoad(({bridge}) => (
-            <PaddedCard>
-              <BridgesForm
-                actionText='Save Bridge'
-                onSubmit={updateBridge}
-                name={bridge.name}
-                nameDisabled
-                url={bridge.url}
-                confirmations={bridge.confirmations}
-                minimumContractPayment={bridge.minimumContractPayment}
-                onSuccess={SuccessNotification}
-                onError={ErrorMessage}
-              />
-            </PaddedCard>
-          ))}
+            <CardContent>
+              {onLoad(({bridge}) => (
```

Comment:
Is there a possibility that `onLoad` returns false? If so, what about accomodating that scenario with a text?